src/main/resources/sql/schema.pgsql

```diff
@@ -39,7 +40,9 @@ CREATE TABLE IF NOT EXISTS orders(
     CREATED_DATE DATE NOT NULL,
     DUE_DATE DATE NOT NULL,
     DELIVERY_LOCATION VARCHAR NOT NULL,
-    ORDER_TYPE VARCHAR NOT NULL
+    ORDER_TYPE VARCHAR NOT NULL,
+    STATUS VARCHAR NOT NULL,
+    ITEMS hstore
```

Comment:
I think hstore will work just fine here, but maybe it's better to use `JSONB` as the data type for the `items` field?

The thing is that we are deserializing this field into a `HashMap` where the values are all strings. If the values are supposed to be item quantities, then it would be nice to have some enforcement that the field contains integers and not only Strings. Can we do that with hstore or is it only strings?

Reply:
Can we do hstore that deserializes to a `Map<String, Integer>` or even to `Map<Integer, Integer>` because the id column is an integer anyways.

Otherwise, i think it is possibly with JSON/JSONB (JSONB best apparently)

Reply:
If we stick with hstore, then we should add some extra processing in the handler function to ensure that valid values are being inserted into the map. Currently, the handler accepts any string.

For example:

```json
"items": {
    "198": "hello",
    "300": "hello"
}
```

is accepted.

Reply:
Side note: we should add some unit tests for the above and similar scenarios, currently the unit tests are not hitting the APIs with different kinds of values like this.

Basically, we are only testing the "success" scenario. We should also test trying to insert bad values